example/example1/example1.go

```diff
@@ -0,0 +1,290 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	openfga "github.com/openfga/go-sdk"
+	"github.com/openfga/go-sdk/client"
+	"github.com/openfga/go-sdk/credentials"
+	"os"
+)
+
+func mainInner() error {
+
+	ctx := context.Background()
+	creds := credentials.Credentials{}
+	if os.Getenv("FGA_CLIENT_ID") != "" {
+		creds = credentials.Credentials{
+			Method: credentials.CredentialsMethodClientCredentials,
+			Config: &credentials.Config{
+				ClientCredentialsClientId:       os.Getenv("FGA_CLIENT_ID"),
+				ClientCredentialsClientSecret:   os.Getenv("FGA_CLIENT_SECRET"),
+				ClientCredentialsApiAudience:    os.Getenv("FGA_API_AUDIENCE"),
+				ClientCredentialsApiTokenIssuer: os.Getenv("FGA_API_TOKEN_ISSUER"),
+			},
+		}
+	}
+
+	apiUrl := os.Getenv("FGA_API_URL")
+	if apiUrl == "" {
+		apiUrl = "http://localhost:8080"
+	}
+	fgaClient, err := client.NewSdkClient(&client.ClientConfiguration{
+		ApiUrl:               apiUrl,
+		StoreId:              os.Getenv("FGA_STORE_ID"),               // not needed when calling `CreateStore` or `ListStores`
+		AuthorizationModelId: os.Getenv("FGA_AUTHORIZATION_MODEL_ID"), // optional, recommended to be set for production
+		Credentials:          &creds,
+	})
+
+	if err != nil {
+		return err
+	}
+
+	// ListStores
```

Comment:
As examples, it will be better to break it into smaller functions so that people can focus on what is needed (i.e., they can comment out things that they don't need